src/main/resources/translations/zh-cn.json

```
@@ -1,100 +1,93 @@
 {
 	"misc.add": "添加",
-	"misc.insert": "插入",
 	"misc.save": "保存",
 	"misc.load": "载入",
 	"misc.remove": "移除",
 	"misc.edit": "编辑",
 	"misc.duplicate": "复制",
 	"misc.error": "错误",
 	"misc.access": "访问",
-	"misc.invalidtype.internal": "格式错误, 应为 'your/class/Name'",
-	"misc.invalidtype.standard": "格式错误, 应为 'Lyour/class/Name;'",
-	
-	"update.fail.resolve": "无法获取自身可执行文件的路径.",
-	"update.fail.jarread": "无法读取内置的 pom.xml.",
-	"update.fail.nodownload": "Recaf 有新版本, 但获取不到下载地址.",
-	"update.fail.unknwon": "更新检查失败: ",
-	"update.outdated": "Recaf 有新版本",
-	"update.complete": "更新完成.",
-	"update.updated": "更新检查完成: 已经是最新版本",
-	"update": "更新",
-	"update.active.name": "允许检查更新",
-	"update.active.desc": "启用后将会按照指定频率定时检查并应用更新.",
-	"update.lastcheck.name": "最后检查",
-	"update.lastcheck.desc": "最后一次检查更新的时间.",
-	"update.frequency.name": "检查频率",
-	"update.frequency.desc": "两次检查之间的最小间隔时间.",
-	"update.frequency.always": "每次启动",
-	"update.frequency.daily": "每天",
-	"update.frequency.weekly": "每周",
-	
-	"display": "显示",
-	"display.simplfy.name": "简化类型描述符",
-	"display.simplfy.desc": "隐藏类型描述符中的包名.",
-	"display.jumphelp.name": "显示跳转语句高亮",
-	"display.jumphelp.desc": "显示操作码中的跳转逻辑.",
-	"display.topmost.name": "置顶编辑窗口",
-	"display.topmost.desc": "编辑时, 将输入窗置顶.",
-	"display.language.name": "语言",
-	"display.language.desc": "界面使用的语言.",
-	"display.style.name": "样式",
-	"display.style.desc": "界面使用的样式表. 资源格式为 'resources/style/NAME-{STYLE}.css'",
-	"display.loglevel.name": "日志级别",
-	"display.loglevel.desc": "控制台中输出的日志级别.",
-	"display.buttonbar.name": "显示快捷工具栏",
-	"display.buttonbar.desc": "在窗体中显示快捷工具栏. 重新启动以应用更改.",
-	"display.maxlength.tree.name": "最大树节点名称长度",
-	"display.maxlength.tree.desc": "文件树中每个节点最多显示多少个字符. 在遇到混淆产生的, 长的吓人的名字时很有用.",
+	"misc.yes": "是",
+	"misc.no": "不",
+	"misc.confirm": "确认",
+	"misc.confirm.message": "你确定吗?",
+
+	"ui.verify": "验证失败",
+	"ui.config": "配置",
+	"ui.history": "历史",
+	"ui.history.pop": "退回到上一个状态",
+	"ui.history.open.class": "打开类",
+	"ui.history.open.file": "打开文件",
+	"ui.info.logging": "日志",
+	"ui.info.other": "其他",
+	"ui.attach": "注入",
+	"ui.attach.copy": "复制到工作空间",
+	"ui.noerrormsg": "未指定错误消息",
```

```suggestion
	"ui.noerrormsg": "无错误消息",
```
这里的意思应该是抛出的 exception 没有 message 部分。